toolchest_client/tools/tool_args.py

```diff
@@ -385,6 +385,11 @@
             "--min-contig-len": 1,
         },
     },
+    "python3": {
```

Comment:
Might be worth blacklisting some args – e.g. `-i` for spawning an interactive Python environment